package.json

```diff
@@ -3,7 +3,7 @@
   "author": "NuoDB, Inc.",
   "description": "The official NuoDB driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
   "license": "Apache-2.0",
-  "version": "3.5",
+  "version": "3.5.0",
```

Comment:
Would prefer this change be marked as 3.5.1

Reply:
Changed to 3.5.1 👍

Reply:
Why would this be 3.5.1 if it is on master? Normally `x.y.0` releases are made on master and `x.y.z` releases are made off of development branches for `x.y`, where `z` is non-0.

It does not make sense to be making patch releases on master. This should be 0.